decayAmplitude/waveDescription.cc

```diff
@@ -63,70 +63,103 @@ using namespace rpwa;
 namespace {
 
 
-	typedef boost::shared_ptr<libconfig::Config> configPtr;
+	typedef boost::shared_ptr<Config> configPtr;
 
 
-	const std::string binBorders = "bins";
-	const std::string expandName = "expand";
-	const std::string expandTypeName = "type";
-	const std::string expandNameName = "name";
+	const string binBorders     = "bins";
+	const string degreeName     = "degree";
+	const string mMinName       = "mMin";
+	const string mMaxName       = "mMax";
+	const string realCoeffName  = "realCoefficients";
+	const string imagCoeffName  = "imagCoefficients";
+	const string expandName     = "expand";
```

Comment:
the functional principle and the nomenclature of variable expansion/binning should be explained in waveDescription.h